src/main/java/com/google/sps/data/TimeRange.java

```diff
@@ -0,0 +1,143 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main.java.com.google.sps.data;
```

Comment:
The others are just `package com.google.sps;`

Reply:
The package structure should match the directory structure.

(In another PR, I commented on the weird choice of using "sps" for the top level (project-specific) package, and requested that _that_ directory and package be renamed. This one here makes sense.